src/js/components/Calendar/stories/Range2.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Box, Calendar, Grommet } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+export const Range = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="large">
+      <Calendar date={[['2020-04-02', '2020-04-08']]} range />
```

Comment:
Instead of creating duplicates of all of the Calendar stories, I think it would be sufficient to just modify one of the existing to demonstrate this new `date` capability.

Reply:
That was just a way of demostrating the working usage with other stories. I will delete them and only make one.